docs/guide/testrunner/configurationfile.md

```diff
@@ -78,7 +78,13 @@ exports.config = {
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
-        browserName: 'chrome'
+        browserName: 'chrome',
+        // to run chrome headless you can use the --headless flag, --disable-gpu is also required for now.
```

Comment:
Then we still have `to run chrome headless` twice. Can we have just one comment about running chrome headless?